src/ui/units/wizard/containers/Wizard/WizardActionPanel/useWizardActionPanel.ts

```diff
@@ -47,6 +49,26 @@ export const useWizardActionPanel = (
 
     const enableEditHistory = Utils.isEnabledFeature(Feature.EnableEditHistory);
 
+    const onClickGoBack = () => {
```

Comment:
`onClickGoBack` and `onClickGoForward` are internally created functions, if we pass this as dependencies to `useMemo` - it will call it every time render is called. So should we use useCallback or save it to ref ourself or don't use useMemo with it.